test/e2e/user_management_test.go

```diff
@@ -556,66 +556,6 @@ func (s *userManagementTestSuite) TestUserDeactivation() {
 }
 
 func (s *userManagementTestSuite) TestUserBanning() {
-	s.Run("ban provisioned usersignup", func() {
-		hostAwait := s.Host()
-		memberAwait := s.Member1()
-		hostAwait.UpdateToolchainConfig(s.T(), testconfig.AutomaticApproval().Enabled(false))
-
-		// Create a new UserSignup and approve it manually
-		user := NewSignupRequest(s.Awaitilities).
-			Username("banprovisioned").
-			Email("banprovisioned@test.com").
-			ManuallyApprove().
-			TargetCluster(memberAwait).
-			RequireConditions(wait.ConditionSet(wait.Default(), wait.ApprovedByAdmin())...).
-			Execute(s.T())
-		userSignup := user.UserSignup
-
-		// Create the BannedUser
-		CreateBannedUser(s.T(), s.Host(), userSignup.Spec.IdentityClaims.Email)
-
-		// Confirm the user is banned
-		_, err := hostAwait.WithRetryOptions(wait.TimeoutOption(time.Second*15)).WaitForUserSignup(s.T(), userSignup.Name,
-			wait.UntilUserSignupHasConditions(wait.ConditionSet(wait.Default(), wait.ApprovedByAdmin(), wait.Banned())...))
-		require.NoError(s.T(), err)
-
-		// Confirm that a MasterUserRecord is deleted
-		_, err = hostAwait.WithRetryOptions(wait.TimeoutOption(time.Second*10)).WaitForMasterUserRecord(s.T(), userSignup.Spec.IdentityClaims.PreferredUsername)
-		require.Error(s.T(), err)
-		// confirm usersignup
-		_, err = hostAwait.WaitForUserSignup(s.T(), userSignup.Name,
-			wait.UntilUserSignupHasConditions(wait.ConditionSet(wait.Default(), wait.ApprovedByAdmin(), wait.Banned())...),
-			wait.UntilUserSignupHasStateLabel(toolchainv1alpha1.UserSignupStateLabelValueBanned))
-		require.NoError(s.T(), err)
-	})
```

Comment:
this was a complete duplication of the test `"ban provisioned usersignup"` below